lib/ui/toolbar.js

```diff
@@ -236,7 +236,8 @@ export default class Toolbar {
 							)}
 						</Select>
 
-						<Button icon="gear" flat="true" disabled={this.state.disableUI || !this.state.codeSigningAvailable} click={this.expandButtonClicked.bind(this)} />
+						<Button title="Compile" icon="gear" flat="true" disabled={this.state.disableUI || !this.state.codeSigningAvailable} click={this.expandButtonClicked.bind(this)} />
+						<Button title="Take screenshot" icon="device-camera" flat="true" disabled={this.state.disableUI} click={this.takeScreenshotButtonClicked.bind(this)}/>
```

Comment:
I'm not sure taking a screenshot is a prominent enough action to have an own - persistent - button in the UI. What do you think about making it like the clean command / flow?

Reply:
probably not. Removed the button and put it into the menu